src/Samples/Samples.sln

```diff
@@ -288,10 +289,10 @@ Global
 		{D4D09B9B-8421-40B9-B4A9-594DAD5AF8DD}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{D4D09B9B-8421-40B9-B4A9-594DAD5AF8DD}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{D4D09B9B-8421-40B9-B4A9-594DAD5AF8DD}.Release|Any CPU.Build.0 = Release|Any CPU
-		{D89124D4-FB1D-4D08-BD95-F60BD24D5965}.Debug|Any CPU.ActiveCfg = Debug|x86
-		{D89124D4-FB1D-4D08-BD95-F60BD24D5965}.Debug|Any CPU.Build.0 = Debug|x86
-		{D89124D4-FB1D-4D08-BD95-F60BD24D5965}.Release|Any CPU.ActiveCfg = Release|x86
-		{D89124D4-FB1D-4D08-BD95-F60BD24D5965}.Release|Any CPU.Build.0 = Release|x86
+		{D89124D4-FB1D-4D08-BD95-F60BD24D5965}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
It appears that these are the TreeTransform samples (which I did not touch). I can't imagine why these would have been `x86` in the `Any CPU` solution configuration to begin with.